data/RI/incentives.json

```diff
@@ -683,5 +683,225 @@
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Rebate of 75% of cost for a Level 2 EV charger putchase where no electrical upgrade is needed, up to $500. Available for income qualified residents."
+    },
+    "start_date": "2024-08-03"
+  },
+  {
+    "id": "RI-48",
+    "payment_methods": [
+      "rebate"
+    ],
+    "items": [
+      "electric_vehicle_charger"
+    ],
+    "program": "ri_oER_powerUpRI",
+    "amount": {
+      "type": "percent",
+      "number": 0.75,
+      "maximum": 1000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Rebate of 75% of cost for purchase and installation of a Level 2 EV charger where an electrical upgrade is needed, up to $1,000. Income-qualified."
+    },
+    "start_date": "2024-08-04"
+  },
+  {
+    "id": "RI-49",
+    "payment_methods": [
+      "rebate"
+    ],
+    "items": [
+      "electric_panel"
+    ],
+    "program": "ri_oER_rhodeIslandHomeElectrificationAndApplianceRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 4000,
+      "maximum": 4000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $4,000 rebate for electrical load service center. Available for low-income residents."
+    }
+  },
+  {
+    "id": "RI-50",
+    "payment_methods": [
+      "rebate"
+    ],
+    "items": [
+      "electric_panel"
+    ],
+    "program": "ri_oER_rhodeIslandHomeElectrificationAndApplianceRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2000,
+      "maximum": 2000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $2,000 rebate for electrical load service center. Available for moderate-income residents."
+    }
+  },
+  {
+    "id": "RI-51",
+    "payment_methods": [
+      "rebate"
+    ],
+    "items": [
+      "electric_panel"
+    ],
+    "program": "ri_oER_rhodeIslandHomeElectrificationAndApplianceRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2500,
+      "maximum": 2500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $2,500 rebate for electric wiring. Available for low-income residents."
+    }
+  },
+  {
+    "id": "RI-52",
+    "payment_methods": [
+      "rebate"
+    ],
+    "items": [
+      "electric_panel"
+    ],
+    "program": "ri_oER_rhodeIslandHomeElectrificationAndApplianceRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1250,
+      "maximum": 1250
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $1,250 rebate for electric wiring. Available for moderate-income residents."
+    }
+  },
+  {
+    "id": "RI-53",
+    "payment_methods": [
+      "rebate"
+    ],
+    "items": [
+      "other"
```

Comment:
Should list the items here: dryer and stove.

It's very awkward to express this in the spreadsheet because Sheets doesn't let you do a multiple-selection dropdown, but the spreadsheet-JSON scripts will actually parse comma-separated values in the Technology column. You have to turn off the data validation on that column in the sheet, manually put in the correct Technology values but comma-separated, then turn validation back on. (E.g. I did this in row 3.)

If you don't want to do all that nonsense, I think it's OK to just do it in JSON manually, and don't bother doing it in the spreadsheet.